stable/keel/templates/_helpers.tpl

```diff
@@ -12,5 +12,9 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 */}}
 {{- define "keel.fullname" -}}
 {{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
```

Comment:
I'd suggest you update to the latest `_helpers.tpl`  you get with `helm create`.